src/client/app/graph/canvas/canvas.component.html

```diff
@@ -4,5 +4,5 @@
      xmlns:xlink="http://www.w3.org/1999/xlink"
      (mousedown)="this.onMousedown($event)"
      (mousewheel)="this.onMousewheel($event)">
-  <g uxg-card *ngFor="let card of this.canvasService.cards" [card]="card"></g>
+  <g uxg-card *ngFor="let card of this.canvasService.cards | async" [card]="card"></g>
```

Comment:
We chatted about this in person. "async" is a pipe that automatically "unwraps" this Observable:

https://angular.io/docs/ts/latest/api/common/index/AsyncPipe-pipe.html